### ISSUE TITLE
Cannot reproduce the demo from the blog post

### ISSUE THREAD

**Issue by benstaf** [bamos/dcgan-completion.tensorflow#10]
I ran the demo from the blog post with 3 images, but I cannot reproduce the result. Instead I get something that does not look reasonable at all:
![completion_newiter2500](https://cloud.githubusercontent.com/assets/8183572/20604739/9a3a5386-b270-11e6-8912-7fed3e33047d.gif)

I even ran it for 2500 iterations. The loss does not go down, it keeps oscillating. checkpoints here:
https://drive.google.com/open?id=0B86WKpvkt66BeEkxeHdmcEVOTVU

Even a non-adaptive optimization technique should give at least reasonable results.
I did not trained any model (a trained model is provided), I only ran the code:

./openface/util/align-dlib.py data/dcgan-completion.tensorflow/data/your-dataset/raw align innerEyesAndBottomLip data/dcgan-completion.tensorflow/data/your-dataset/aligned --size 64

./complete.py ./data/your-test-data/aligned/* --outDir outputImages

cd outputImages convert -delay 10 -loop 0 completed/*.png completion.gif

Did I miss a step in the blog post?

I use tensorflow 0.11 on a CPU on AWS EC2  c4.4xlarge

**Comment by bamos** [comment]
Hi, can you try lowering the learning rate and turning off momentum?

-Brandon.

**Comment by benstaf** [comment]
By dividing the learning rate by 100, and turning off momentum, the loss function decreases better. However, the result is still bad (especially the first picture, which is a selfie from myself, while the other 3 are famous politicians):

![completion_newbushrumsfeld](https://cloud.githubusercontent.com/assets/8183572/20711462/78fd393c-b648-11e6-9153-b44da8067ddf.gif)


In your demo, what is the final loss?
I could not get less than 1500.

I ran the command:

./complete.py /home/ubuntu/dcgan-completion.tensorflow/static/aligned/rawfaces/* --lr 0.0001 --momentum 0.0   --nIter 1200 --outDir /home/ubuntu/dcgan-completion.tensorflow/static/outputimages


checkpoints here:
![checkpoints](https://cloud.githubusercontent.com/assets/8183572/20711430/4fac6f1c-b648-11e6-8e5f-c9450baf926c.png)

**Comment by bamos** [comment]
The last three there look much better than before, an adaptive technique would definitely help here. An adaptive one could potentially help the first image so you don't have to tune the learning rate. You could also try changing the `lambda` weight. I don't remember the final loss I got.

I'll close this issue for now but let me know if you notice anything else.

-Brandon.
